src/basic_commands/zoom.md

```diff
@@ -1,82 +0,0 @@
-## Zoom
```

Comment:
this was deleted, be sure to remove the link also from the book.

Reply:
done